scenes/game/local_game.gd

```diff
@@ -933,6 +935,36 @@ class Player:
 			deck.insert(0, card)
 		return events
 
+	func get_unknown_cards():
+		var unknown_cards = hand + deck # TODO: secret sealed areas
```

Comment:
don't forget this

Reply:
This is mostly a reminder for when we implement a character that has a secret sealed area (specter/arakune)